pre-commit/notebook_urls.py

```diff
@@ -110,6 +110,10 @@ def main(files: List[Path], from_ref: str, to_ref: str) -> bool:
     passed = True
     print(f"From ref '{from_ref}' to ref '{to_ref}'")
     for path in files:
+        if path.suffix != ".ipynb":
+            print(f"Skipping {path} (not a Jupyter Notebook).")
+            return passed
+
```

Comment:
The file extension does not make it a jupyter notebook. We should not rely on this.